WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -99,6 +107,15 @@ class WPMainActivityViewModel @Inject constructor(
             .distinctUntilChanged()
             .map { Event(it) } as LiveData<Event<List<FocusPointInfo>>>
 
+    val isMultipleSiteAvailable: Boolean
```

Comment:
Can be renamed to `hasMultipleSitesAvailable`. Wdyt?. 🤔

Reply:
It makes sense. I have renamed to `hasMultipleSites`. Done in [commit](https://github.com/wordpress-mobile/WordPress-Android/pull/16063/commits/8bcb6feea611ad7bc7154d9dd89b72c73efe2c3b).